tensorflow/core/kernels/conv_grad_filter_ops.cc

```diff
@@ -838,11 +838,11 @@ void LaunchConv2DBackpropFilterOp<Eigen::GpuDevice, T>::operator()(
   VLOG(3) << "Compute Conv2DBackpropFilter with cuDNN:"
           << " data_format=" << ToString(data_format)
           << " compute_data_format=" << ToString(compute_data_format);
-
-  constexpr auto kComputeInNHWC =
+  
+  auto kComputeInNHWC =
```

Comment:
Is there any way we can keep the constexpr here?

Reply:
I don't know. The AppleClang compiler thinks returned values should not be constant.

Reply:
I figured it out. Simply replace`constexpr` with `const` will solve the problem. According to C++11 docs, `constexpr` must be determined at compilation time, and `const` can be determined at run time.